### PR TITLE
RHEL 8.3 has KRB5 1.18 with KDB 8.0

### PR DESCRIPTION
**Pull request by tiran**
Signed-off-by: Christian Heimes <cheimes@redhat.com>

### CODE REVIEW

**Comment by fcami** [comment]
LGTM. Will wait for CI before ACK.

File: freeipa.spec.in [hunk 1]
```diff
@@ -46,8 +46,8 @@
 %if 0%{?rhel}
 %global package_name ipa
 %global alt_name freeipa
-%global krb5_version 1.16.1
-%global krb5_kdb_version 7.0
+%global krb5_version 1.18
+%global krb5_kdb_version 8.0
```
Comment (fcami) [inline freeipa.spec.in:50]: LGTM - will wait for CI before ACK.

**Comment by abbra** [comment]
May be instead of cherry-picking small changes, do a single PR that combines spec file updates from both Fedora Rawhide and RHEL 8.3? There are more changes than this one.

**Comment by fcami** [comment]
@tiran please see @abbra 's comment above, but that's up to you.

**Comment by flo-renaud** [comment]
master:

* 1144da5d94aa75e882929e136d7d322dc4a93732 RHEL 8.3 has KRB5 1.18 with KDB 8.0

**Pull request closed without merging**